recipes/erlang/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export LIBRARY_PATH="${PREFIX}/lib:${LIBRARY_PATH}"
+export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+export ERL_TOP="$(pwd)"
+./configure --with-ssl="${PREFIX}" --prefix="${PREFIX}" --without-javac
```

Comment:
Ok, well, I added it before I got this message. Let's see if this works. If not, then we can always remove it and proceed without atomic ops for the time being.